### PR TITLE
Feat/toggle bool css

### PR DESCRIPTION
**Pull request by joseglego**
# Changes
- Change from toggle slide image to CSS.
- Add visual effect for switching.

**After**
image

**Before**
toggle-bool-column

### CODE REVIEW

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+      display: inline-block;
+      width: 20px;
+      height: 12px;
+      position: absolute;
+      left: 3px;
+      top: 2px;
+
+      -webkit-transition: all .25s linear;
+      transition: all .25s linear;
+      transform: translateX(0px);
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected unit                                 length-zero-no-unit

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+      box-shadow: $box-shadow;
+
+      display: inline-block;
+      width: 20px;
+      height: 12px;
+      position: absolute;
+      left: 3px;
+      top: 2px;
+
+      -webkit-transition: all .25s linear;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected vendor-prefix "-webkit-transition"   property-no-vendor-prefix

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss [hunk 3]
```diff
@@ -1,11 +1,53 @@
+$inactive-bg: #6a7176;
+$inactive-shadow: #4d5256;
+$active-bg: #4dbcff;
+$active-shadow: #2f749e;
+$button-bg: #fff;
+$button-shadow: #dfe1e2;
+
+$box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 0 1px #fff inset;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected leading zero                         number-leading-zero

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss (hunk 1 above)
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected unit                                 length-zero-no-unit

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss (hunk 2 above)
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected vendor-prefix "-webkit-transition"   property-no-vendor-prefix

File: app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss (hunk 3 above)
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/addons/toggle_bool.scss]: Unexpected leading zero                         number-leading-zero

**Comment by joseglego** [comment]
@felbalart listo 👍

**Pull request merged**